app/views/issues/show.html.erb

```diff
@@ -35,8 +35,8 @@
 <!-- Main content -->
 <div class="row m-0">
   <ul class="tabs-container nav nav-tabs main-tabs w-100">
-    <li class="nav-item"><a href="#info-tab" data-toggle="tab" class="nav-link active"><%= colored_icon_for_model(@issue, 'fa-bug') %> Information</a></li>
-    <li class="nav-item"><a href="#evidence-tab" class="nav-link" data-toggle="tab"><i class="fa fa-laptop"></i> Evidence <span class="badge bg-primary text-white"><%= @issue.evidence.count %></span></a></li>
+    <li class="nav-item"><a href="#info-tab" data-toggle="tab" class="nav-link <%= class_names(active: !params[:tab]) %>"><%= colored_icon_for_model(@issue, 'fa-bug') %> Information</a></li>
```

Comment:
I'm not sure any of the references to `params[:tab]` belong to this PR.

Afaict, it's only used in this view, and only to render HTML, can you please remove?

Reply:
This is setting us up for the bulk delete pr and adding new evidence pr, so that those actions return the user to the evidence tab.